infra/cifuzz/fuzz_target.py

```diff
@@ -72,7 +73,9 @@ def fuzz(self):
     command += [
         '-e', 'FUZZING_ENGINE=libfuzzer', '-e', 'SANITIZER=address', '-e',
         'RUN_FUZZER_MODE=interactive', 'gcr.io/oss-fuzz-base/base-runner',
-        'bash', '-c', 'run_fuzzer {0}'.format(self.target_name)
+        'bash', '-c', 'run_fuzzer {fuzz target} {seed} {len control}'.format(self.target_name,
```

Comment:
I don't think it works this way, the arguments passed to the `format()` function all should be named (i.e. `.format(fuzz_target=self.target_name, ...)`

Also, I wasn't explicit on that in my previous comment, let's put both options into a single `options` parameter.

The constant string `-seed=1337 -len_control=0` can go into some global constant (e.g. `DEFAULT_OPTIONS = '-seed=1337 -len_control=0'`) and we may want to make it possible to modify in future

Reply:
That makes a lot of sense, thanks.